### PR TITLE
Change "mame64" to "mame" in docs

### PR DESCRIPTION
**Pull request by Firehawke**
Need a specific review on compilingmame.rst line 205, though.

### CODE REVIEW

**Review by Firehawke — COMMENTED** [review]
```
It is not possible to cross-compile a 32-bit version of MAME using 64-bit MinGW
tools on Windows, the 32-bit MinGW tools must be used.  This causes issues due
to the size of MAME.  It is not possible to link a full 32-bit MAME build
including the SDL OS-dependent layer and the Qt debugger.  GNU ld and lld will
both run out of memory, leaving an output file that doesn’t work.  It’s also
impossible to make a 32-bit build with full local variable symbols.  GCC may run
out of memory, and certain source files may exceed the limit of 32,768 sections
imposed by the PE/COFF object file format.
```

Is this still true as of now? @Tafoid said he's been doing compiles without issues, so I want to be 100% sure before I make any changes here.

File: docs/source/initialsetup/compilingmame.rst [hunk 1]
```diff
@@ -71,7 +71,7 @@ building MAME on a 64-bit system.  Instructions may need to be adjusted for
   window management, audio/video output, font rendering, etc.  If you want to
   use the portable SDL (Simple DirectMedia Layer) interfaces instead, you can
   add **OSD=sdl** to the make options.  The main emulator binary will have an
-  ``sdl`` prefix prepended (e.g. ``sdlmame64.exe`` or ``sdlmame.exe``).  You
+  ``sdl`` prefix prepended (e.g. ``sdlmame.exe`` or ``sdlmame.exe``).  You
```
Comment (cuavas) [inline docs/source/initialsetup/compilingmame.rst:74]: I think  you forgot to read this one – sdlmame or sdlmame.

File: docs/source/commandline/commandline-all.rst [hunk 2]
```diff
@@ -63,20 +63,18 @@ executable (for example you could call it ``mame-here`` )::
 
     #!/bin/sh
     cd "`dirname "$0"`"
-    exec ./mame64
+    exec ./mame
 
 You should be able to use any text editor.  If you have a choice of file format
-or line ending style, choose UNIX.  This assumes you're using a 64-bit release
-build of MAME, but if you aren't you just need to change ``mame64`` to the name
```
Comment (cuavas) [inline docs/source/commandline/commandline-all.rst:70]: You can still end up with `mamed`, `mamep`, `mamedp` etc. depending on other options – should the note about possibly needing to change it be left in and just adjusted?

**Comment by cuavas** [comment]
> 
> 
> ```
> It is not possible to cross-compile a 32-bit version of MAME using 64-bit MinGW
> tools on Windows, the 32-bit MinGW tools must be used.  This causes issues due
> to the size of MAME.  It is not possible to link a full 32-bit MAME build
> including the SDL OS-dependent layer and the Qt debugger.  GNU ld and lld will
> both run out of memory, leaving an output file that doesn’t work.  It’s also
> impossible to make a 32-bit build with full local variable symbols.  GCC may run
> out of memory, and certain source files may exceed the limit of 32,768 sections
> imposed by the PE/COFF object file format.
> ```
> 
> Is this still true as of now? @Tafoid said he's been doing compiles without issues, so I want to be 100% sure before I make any changes here.

Yes, it’s still true.  You’ll hit the PE/COFF section limit on the emumem files if you try building with full local variable name symbols, and we’re really close to the memory limit for the linkers.

File: docs/source/initialsetup/compilingmame.rst (hunk 1 above)
Comment (Firehawke) [reply docs/source/initialsetup/compilingmame.rst:74]: Yeah. Oops. Fixed.

File: docs/source/commandline/commandline-all.rst (hunk 2 above)
Comment (Firehawke) [reply docs/source/commandline/commandline-all.rst:70]: Adjusted this, let me know if that meets your expectations.

**Pull request merged**